src/extensionStore.js

```diff
@@ -79,12 +79,14 @@ class ExtensionStore {
     }
 
     get directories() {
-        let configuration = vscode.workspace.getConfiguration("private-extension-manager")
+        let configuration = vscode.workspace.getConfiguration("private-extension-manager");
+
+        const rootDir = vscode.workspace.workspaceFolders[0].uri.fsPath;
```

Comment:
This means that we don't support multi-root workspaces, but I think it's ok for now? Otherwise I'm not sure how to pick the root path